_bmad-output/planning-artifacts/ux-design-specification.md

```
@@ -0,0 +1,111 @@
+- `dropSubscriberProcedureResultMsg` handles completion and resets `dropProcedureDeleting`
+
+---
+
+## Acceptance Criteria
+
+1. Pressing `P` (not `Ctrl+D`) opens the delete confirmation
+2. Pressing `Ctrl+D` while on Database Settings does nothing
+3. After confirming delete, the Danger Zone shows spinner + wait message
+4. Success/error message appears **inline** within the Danger Zone after deletion completes
+5. The result message is cleared on `Esc` or panel close
+6. All changes pass `make build` and `make test`
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)